m2c/asm_file.py

```diff
@@ -528,7 +530,13 @@ def process_label(label: str, *, kind: LabelKind) -> None:
                         for w in args:
                             fval = try_parse(lambda: float(w))
                             asm_file.new_data_bytes(struct.pack(">d", fval))
-                    elif directive in (".asci", ".asciz", ".ascii", ".asciiz"):
+                    elif directive in (
+                        ".asci",
+                        ".asciz",
+                        ".ascii",
+                        ".asciiz",
+                        "string",
```

Comment:
This won't work -- this section is guarded by `directive.startswith(".")`

Reply:
My bad, it should actually be ".string"